1-exercises/D-for-in-loops/exercise-2.js

```diff
@@ -25,6 +25,20 @@ let shapesWithMoreThanFiveSides = [];
 
 // WRITE YOUR CODE BELOW THIS LINE
 
+for (let shape in shapes) {
```

Comment:
The solution is incomplete. You should be adding the elements into ``shapesWithMoreThanFiveSides`` array